tests/test_examples.py

```diff
@@ -0,0 +1,122 @@
+import json
+
+from tests.conftest import FlowHeaterLayer
+from tests.util import execute
+
+
+def test_error_syntax(fh_http: FlowHeaterLayer):
+    """
+    $ deno run examples/01-basic/error-syntax.js
+    error: Expected ';', '}' or <eof> at file:///Users/amo/dev/flow-heater/fh-core/examples/01-basic/error-syntax.js:1:6
+    """
+
+    response = execute("01-basic/error-syntax.js", headers={"FH-Debug": "true"})
+
+    assert response.status_code == 500
+    data = response.json()
+
+    # TODO: That might yield a linter error after the HTTP request
+    #       header "FH-Debug: true" will get evaluated appropriately.
+
+
+def test_error_runtime(fh_http: FlowHeaterLayer):
+    """
+    $ deno run examples/01-basic/error-runtime.js
+    error: Uncaught TypeError: JSON.foobar is not a function
+    JSON.foobar();
+         ^
+        at file:///Users/amo/dev/flow-heater/fh-core/examples/01-basic/error-runtime.js:1:6
+    """
+
+    response = execute("01-basic/error-runtime.js", headers={"FH-Debug": "true"})
+
+    assert response.status_code == 500
+    data = response.json()
+
+    # TODO: That might yield a stacktrace after the HTTP request
+    #       header "FH-Debug: true" will get evaluated appropriately.
+
+
+def test_json_demo(fh_http: FlowHeaterLayer):
+
+    response = execute("01-basic/json-demo.js")
```

Comment:
Deluxe API design. Thanks a lot!

Reply:
`execute()` takes another fixed kwarg `method` and all the other `**kwargs` will get forwarded to the `requests` call to be able to stuff the whole shebang of request and header parameters into it.

Reply:
> the whole shebang of request and header parameters into it.

Currently, in `test_examples.py` we use `data`, `json` and `headers` already.